ci/build_docs.sh

```diff
@@ -52,18 +54,23 @@ pushd cpp/doxygen
 doxygen Doxyfile
 mkdir -p "${RAPIDS_DOCS_DIR}/libcugraph/html"
 mv html/* "${RAPIDS_DOCS_DIR}/libcugraph/html"
+mkdir -p "${RAPIDS_DOCS_DIR}/libcugraph/_xml"
+mv xml/* "${RAPIDS_DOCS_DIR}/libcugraph/_xml"
 popd
 
 rapids-logger "Build Python docs"
 pushd docs/cugraph
 # Ensure cugraph is importable, since sphinx does not report details about this
 # type of failure well.
 python -c "import cugraph; print(f'Using cugraph: {cugraph}')"
-sphinx-build -b dirhtml source _html
-sphinx-build -b text source _text
+sphinx-build -v -b dirhtml source _html
+sphinx-build -v -b text source _text
```

Comment:
```suggestion
sphinx-build -b dirhtml source _html
sphinx-build -b text source _text
```

We should remove these before the merge to prevent excessive log output.